Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3790,34 +3790,6 @@ LayoutViewportHeightExpansionFactor:
     WebCore:
       default: 0
 
-LazyIframeLoadingEnabled:
-  type: bool
-  status: stable
-  category: html
-  humanReadableName: "Lazy iframe loading"
-  humanReadableDescription: "Enable lazy iframe loading support"
-  defaultValue:
-    WebKitLegacy:
-      default: true
-    WebKit:
-      default: true
-    WebCore:
-      default: true
-
-LazyImageLoadingEnabled:
-  type: bool
-  status: stable
-  category: html
-  humanReadableName: "Lazy image loading"
-  humanReadableDescription: "Enable lazy image loading support"
-  defaultValue:
-    WebKitLegacy:
-      default: false
```

Comment:
This feature isn't enabled in WebKitLegacy. Why is it okay to enable this feature in WebKitLegacy now?

Reply:
Hmm, indeed. So why is iframe loading not also restricted? Surely it also relies on Intersection Observer? cc @rwlbuis